### DISCUSSION TITLE
Import File Formats (CSV & TSV) - What can I import?

### DISCUSSION THREAD

**Discussion by spirited1086** [FreeLanguageTools/vocabsieve#62] [Q&A] [answered]
I just saw CSV and TSV are listed as additional formats from which I can import files? What  is Vocabsieve expecting with these types of files? What am I able to import with them? A frequency list? A known word list?

**Comment by 1over137** [comment, marked as answer]
You can import dictionaries, with the first column being the word, second column being definition. This is mostly a convenience feature for those not able to work with proper dictionary formats but only have access to raw table data.
